benchmark/single-source/Array2D.swift

```diff
@@ -15,10 +15,11 @@ import TestsUtils
 public let Array2D = BenchmarkInfo(
   name: "Array2D",
   runFunction: run_Array2D,
-  tags: [.validation, .api, .Array])
+  tags: [.validation, .api, .Array],
+  setUpFunction: { blackHole(Arr) })
```

Comment:
I think we should store these workload data structures in `Optional`s, so that we can release them in the `tearDownFunction`. Otherwise we're effectively leaking memory, as we go through the pre-commit benchmark suite. This single array here is 8MB.